lib/facter/sssd.rb

```diff
@@ -1,19 +1,6 @@
 require 'facter/util/sssd'
 
 if defined? Facter::Util::Sssd
-  # == Fact: foreman_ipa
-  Facter.add(:foreman_ipa, :type => :aggregate) do
-    {
-      :default_realm => 'global/realm',
```

Comment:
I might end up needing the realm for Debian support 🙈

Reply:
With https://github.com/theforeman/foreman-documentation/pull/594 I think we can safely remove it? If so, I'll rebase this.

Reply:
Yes, that was my intention!